### ISSUE TITLE
All-Sigs roadmap meeting for end of 30th of May 2023

### ISSUE THREAD

**Issue by lawsonamzn** [o3de/tsc#80]
8AM Pacfic Time (US) in the TSC/TAC discord channel, will be the monthly "all sigs" status update and roadmap on 5/30/2023 (TUESDAY)

See the previous agenda for an example:
 https://github.com/o3de/tsc/issues/74

Please attach links to the SIG roadmap (And anything to notify teams of in case of your absence, it will be read for you), in comments below. along with any agenda items.

**Comment by Naomi-Wash** [comment]
@lawsonamzn Can we update the date to 30 May which is Tuesday?

## Suggested Agenda Items: 

### Master Roadmap 

- [Can we restart and build on this](https://github.com/o3de/o3de/projects/1)? 
  - Changing/adding columns? (example: Goals, Possible Features of xx Release)

**Comment by vincent6767** [comment]
SIG Release roadmap: https://github.com/orgs/o3de/projects/29

What we did last month
1. Released 23.05 in collaboration with other SIGs and gathered release feedback from the team.

What we're going next month based on the roadmap: https://github.com/orgs/o3de/projects/29
1. Do the 23.05 release retrospective, and decide on the next item of improvement.
2. Together with TSC to decide whether we would want to mention the roadmap maintenance are one of the SIG responsibilities on the SIG charter. This is the issue that tracks this discussion: https://github.com/o3de/sig-release/issues/146
3. Start a discussion about the next release date.

Proposed Agenda
1. Decides the next steps for mentioning the roadmap maintenance responsibility in the SIG charter (or general guidelines).
2. To discuss features grid maintenance effort and make sure everyone agrees that it's important to be maintained.
